vendor/bat-native-ledger/src/bat/ledger/internal/media/twitter.cc

```diff
@@ -216,22 +216,28 @@ std::string MediaTwitter::GetShareURL(
   auto comment = args.find("comment");
   auto name = args.find("name");
   auto tweet_id = args.find("tweet_id");
+  auto hashtag = args.find("hashtag");
   if (comment == args.end() || name == args.end() || tweet_id == args.end())
     return std::string();
 
+  // Append hashtag to comment ("%20%23" = percent-escaped space and
+  // number sign)
+  std::string comment_with_hashtag =
```

Comment:
could you please add unit tests for `GetShareURL`

Reply:
Yes, done.